desktop/apps/artwork/components/commercial/templates/index.jade

```diff
@@ -1,5 +1,5 @@
 .artwork-commercial( class='js-artwork-commercial' )
-  form.artwork-commercial__form
+  form.artwork-commercial__form( style= !artwork.is_for_sale ? "padding-top: 0" : "" )
```

Comment:
Right, but the question section wouldn't be shown unless the `artwork.is_inquireable` is true.